aiohttp/web_fileresponse.py

```diff
@@ -48,6 +48,25 @@
     {ext: mimetypes.encodings_map[ext] for ext in (".br", ".gz")}
 )
 
+FALLBACK_CONTENT_TYPE = "application/octet-stream"
+
+# Provide additional MIME type/extension pairs to be recognized.
+# https://en.wikipedia.org/wiki/List_of_archive_formats#Compression_only
+ADDITIONAL_CONTENT_TYPES = MappingProxyType(
+    {
+        "application/gzip": ".gz",
+        "application/x-brotli": ".br",
+        "application/x-bzip2": ".bz2",
+        "application/x-compress": ".Z",
+        "application/x-xz": ".xz",
+    }
+)
+
+# Add custom pairs and clear the encodings map so guess_type ignores them.
+mimetypes.encodings_map.clear()
```

Comment:
We need a https://docs.python.org/3/library/mimetypes.html#mimetypes.MimeTypes here to operate on so we don’t affect any other packages using mimetypes

Reply:
We're already modifying it by adding Brotli for < 3.9.  Does that need to be part of the custom object too?

Reply:
Well, 3.8 will be dropped shortly, so I wouldn't waste any time on it.